BGPViewerCore/Service/BGPHeService.cs

```diff
@@ -194,7 +194,27 @@ public IEnumerable<IxModel> GetAsnIxs(int asNumber)
 
         public Tuple<IEnumerable<AsnModel>, IEnumerable<AsnModel>> GetAsnPeers(int asNumber)
         {
-            throw new NotImplementedException();
+            var driver = GetDriverWithValidatedResponseFrom($"https://bgp.he.net/AS{asNumber}");
+            var hasIPv4Peers = driver.PageSource.Contains("table_peers4");
+            var hasIPv6Peers = driver.PageSource.Contains("table_peers6");
+            
+            var ipv4Peers = hasIPv4Peers ? ExtractAsnsInfoFromPeerTable(driver.FindElement(By.Id("peers")))
+                .Select(x => new AsnModel{
+                    ASN = int.Parse(x[0]),
+                    Name = x[1],
+                    Description = x[2],
+                    CountryCode = null
+                }) : new AsnModel[]{};
+                
+            var ipv6Peers = hasIPv4Peers ? ExtractAsnsInfoFromPeerTable(driver.FindElement(By.Id("peers6")))
```

Comment:
This line should reference `hasIPv6Peers` and not `hasIPv4Peers`. It will cause problems if searched AS doesn't have IPv6 peers.